apps/i18n/common/en_us.json

```diff
@@ -1210,6 +1210,7 @@
   "keyValuePairsHeader": "KEY/VALUE PAIRS",
   "keyValuePairLink": "Key/value pairs",
   "kindergarten": "Kindergarten",
+  "labContainerSlowLoading": "This is taking longer than usual...",
```

Comment:
Should this just be `slowLoading`? That would match the server-side string key and could potentially be reused

Reply:
Fair call.  Will update.